packages/react/import-component/index.js

```diff
@@ -0,0 +1,85 @@
+/* global __webpack_modules__, __webpack_require__ */
+const { createElement, Component, useContext } = require('react');
+const PropTypes = require('prop-types');
+const deprecate = require('depd')('hops-react');
+const ImportComponentContext = require('./context');
+
+exports.importComponent = ({ load, moduleId }, name = 'default') => {
+  const resolve = typeof name === 'function' ? name : (module) => module[name];
+
+  if (typeof name === 'string' && name !== 'default') {
+    deprecate(
+      '[DEP002] Using a string to resolve the module of `importComponent` is deprecated and will be removed in a future major release (https://github.com/untool/untool/blob/master/DEPRECATIONS.md).'
```

Comment:
`DEP002` (actually `DEP0002`…) is currently used for the `hops-express` preset, which can be removed in favor of this one.

Also the URL to the `DEPRECATIONS.md` is wrong.

Reply:
See https://github.com/xing/hops/pull/1164#discussion_r412828641